.circleci/config.yml

```diff
@@ -14,7 +14,7 @@ jobs:
 
   build-and-test:
     macos:
-      xcode: 12.0.1 # Specify the Xcode version to use
+      xcode: 13.4.0 # Specify the Xcode version to use
```

Comment:
Is the code coverage issue fixed in 13.4?

Reply:
I'm not seeing an issue with CodeCov. I've updated the version because version 12 of Xcode is no longer supported in CircleCI. The builds just fail now. Their documentation states that they support the latest 4 major.minor versions of Xcode. We've used Xcode 13.0.0 for other projects, but that is the 4th version. If there is a 13.5 version of Xcode, then 13.0.0 may not be supported. I didn't want to bump all the way to Xcode 14, so I picked that latest Xcode 13 image supported by CircleCI.